### ISSUE TITLE
Fix representation issue of highway=bus_guideway

### ISSUE THREAD

**Issue by Lemalic** [cyclosm/cyclosm-cartocss-style#318]
In my city we have some lines of guided bus, shared with bicycles (very useful). Those lines appear on OSM but vanish on CyclOSM.
Is it possible to keep them on the layer when their access is open to bicycles?
Pictures joined to show the problem (on OSM look for "Nîmes, France").
![Bus_guideway_OSM](https://user-images.githubusercontent.com/17762350/75764288-eedbec80-5d3d-11ea-8521-bc784ef09244.png)
![Bus_guideway_CyclOSM](https://user-images.githubusercontent.com/17762350/75764285-ee435600-5d3d-11ea-9964-3424667fc421.png)

**Comment by Phyks** [comment]
Indeed, thanks for the feedback! Quick thought, this might be rendered as a shared cycleway / busway for example.

**Comment by Florimondable** [comment]
I’m not sure it’s well tagged, it’s look like a regular bus lane for me, it seems that bus_guideway should be used where bus are like in "guided rails" where other vehicules can’t go

**Comment by Lemalic** [comment]
It is indeed difficult to tag them properly, but it is clearly not a regular bus lane. We locally call them "Tram'bus". Buses are partly optically guided, they have priorities in all traffic lights turning green when they come thanks to radio connection which ensure "cadence".
Their lanes are shared with other motorised vehicle only on few narrow parts of the city
Shared busway would be a good rendering for me.

**Comment by Lemalic** [comment]
Here is what I found on wikipedia on this king of buses
https://en.wikipedia.org/wiki/Bus_rapid_transit

**Comment by Phyks** [comment]
The place has been updated in OSM since the first post of this issue: https://www.cyclosm.org/#map=19/43.83420/4.36028/openstreetmap.

We still miss the `highway=bus_guideway` which can help locate oneself. What about rendering them the same as a tramway railway? (from wiki pictures, it seems to look similar on the ground, https://wiki.openstreetmap.org/wiki/Tag:highway%3Dbus_guideway).

@Florimondable I thought we had some example of tramway rails which were open to bicycles, but I cannot find them. Do you have that in mind?

**Comment by Florimondable** [comment]
@Phyks Yes I think we can render them the same way as tramway rails.

The only bus_guideway with bicycle=yes are in France and I really think it's bad tagging. For the example, the one in Nîmes is for me just a bus lane.

**Comment by Lemalic** [comment]
@Phyks you might think about Besançon ?

 which seems to have several streets shared Tramway / cars / bikes. Quai Vieil Picard, Avenue Fontaine Argent
![Besancon FontaineArgent](https://user-images.githubusercontent.com/17762350/79384071-8dc34d80-7f66-11ea-9c46-5e595fe2d3af.png)

![Besancon VeilPicard](https://user-images.githubusercontent.com/17762350/79384203-bd725580-7f66-11ea-9bbb-edf70598624b.png)

**Comment by Lemalic** [comment]
@Florimondable clearly it's not a normal bus lane on most of its way. Few pics from Google StreetView to demonstrate:
- Here the bus can not use the "Tram'bus" lane (guided_busway)
![Trambus1](https://user-images.githubusercontent.com/17762350/79384563-691ba580-7f67-11ea-9e19-a82084e589b2.png)

- Here the dot lines in the middle of the lane serve to guide the bus
![Trambus2](https://user-images.githubusercontent.com/17762350/79384874-f3640980-7f67-11ea-9eff-b4f74cb81410.png)


- And there you can clearly see the "Tram'bus" shared with bikes
![Trambus3](https://user-images.githubusercontent.com/17762350/79385028-3625e180-7f68-11ea-8495-08d6bcad5f94.png)


I am not sure the "guided_busway" is well tag all along (maybe we should create a specific line all along the way even when its not physically separate from the car highway like on this last pic ?); but I am sure it's not a basic bus lane.

**Comment by Phyks** [comment]
Maybe about Besançon. But the tagging there is railway + separate way with railway tags, which seems correct in CyclOSM (https://www.cyclosm.org/#map=18/47.24518/6.03230/cyclosm). From the tagging only, I'm not discussing whether this is well tagged or not.

Do you have OSM links to the locations of the Google pictures in Nimes to have a look at the tagging around? Thanks!

**Comment by Lemalic** [comment]
I thought you were looking for places where bikes and tramway share the same lane... It seems it is the case in several streets of Besançon.

Anyway, the location in Nîmes is there: https://www.openstreetmap.org/#map=19/43.82577/4.35394
